src/main/resources/public/js/components/forms/organization-invitation-form.js

```diff
@@ -39,86 +42,156 @@ class OrganizationInvitationForm extends React.Component {
+            this.csvReader.cleanInput();
+        }
+    };
+
     onSubmit(e) {
         e.preventDefault();
+        const {email, admin, emailsFromCSV} = this.state;
 
-        this.setState({isLoading: true});
-        this.props.createOrganizationInvitation(this.props.organization.id, this.state.email, this.state.admin)
-            .then(() => {
-                this.setState({
-                    email: '',
-                    admin: false,
-                    isLoading: false,
-                    error: '',
-                    success: this.context.t('ui.request.send')
-                });
-            })
-            .catch((err) => {
-                this.setState({
-                    isLoading: false,
-                    error: err.error
+
+        if (emailsFromCSV.length > 0) {
+            this._inviteMultipleUsers(emailsFromCSV);
+        }
+
+        if(email !== '') {
+            this.setState({isLoading: true});
+            this._organizationService.inviteUser(this.props.organization.id, email, admin)
+                .then((res) => {
+                    this.props.callBackMembersInvited(res);
+                    this.setState({
+                        email: '',
+                        admin: false,
+                        isLoading: false,
+                        error: '',
+                        success: this.context.t('ui.request.send')
+                    });
+                })
+                .catch((err) => {
+                    this.setState({
+                        isLoading: false,
+                        error: err.error
+                    });
                 });
-            });
+        }
     }
 
     render() {
+        const {csvLoading, isFetchingUsers, emailsFromCSV, isLoading, email} = this.state;
+        const submitButton =
+            <button type="submit" className="btn btn-submit" disabled={(isFetchingUsers || csvLoading || isLoading)}>
+                {this.context.t('my.network.invite-user')}
+            </button>;
+        const requiered = !(emailsFromCSV.length > 0 || email !== '');
+
         return <header className="dropdown-header">
             <form className="organization-invitation-form flex-row" onSubmit={this.onSubmit}>
-            <fieldset className="flex-col">
-                <div className="flex-row-mobile-column">
-                    <div className="flex-row wrapper">
-                        <div className="flex-row">
-                            <label className="label">
-                                {this.context.t('organization.form.email')}
-                                <input name="email" type="email" className="field form-control no-auto" required={true}
-                                       onChange={this.handleChange} value={this.state.email}/>
-                            </label>
+                {/*email input*/}
+                <div className={"form-row-block"}>
```

Comment:
why are you wrapping the className values inside {} ?

Reply:
That is an habit, if I want to add a dynamic class the {} are here, and in my opinion that keeps the code homogeneous.